pip/download.py

```diff
@@ -399,11 +399,6 @@ def get_file_content(url, comes_from=None, session=None):
     :param comes_from:  Origin description of requirements.
     :param session:     Instance of pip.download.PipSession.
     """
-    if session is None:
-        raise TypeError(
-            "get_file_content() missing 1 required keyword argument: 'session'"
```

Comment:
I think it's nicer that `get_file_content` would fail if it's not passed `session`. I don't have to think about what the function does when using it.